src/main/resources/messages_de.properties

```diff
@@ -711,6 +711,9 @@ settings.overtime.description=Die maximale Anzahl der Überstunden bestimmt, wie
 settings.overtime.overtimeActive=Überstundenverwaltung aktivieren
 settings.overtime.overtimeActive.true=aktivieren
 settings.overtime.overtimeActive.false=deaktivieren
+settings.overtime.overtimeReductionWithoutApplicationActive=Überstundenabbau ohne Antrag aktivieren
+settings.overtime.overtimeReductionWithoutApplicationActive.true=aktivieren
+settings.overtime.overtimeReductionWithoutApplicationActive.false=deaktivieren
```

Comment:
bin überhaupt nicht zufrieden mit `aktivieren / deaktivieren`

das ist eher ein `erlauben und ja/nein`. müsste man das dann bei allen Einstellungen anpassen? Mal noch überlegen.

Reply:
Meine Idee war da immer wie lassen das auf aktivieren / deaktivieren bis wir eine neue UI haben und machen das dann in em Zuge. Finde aktivieren / deaktivieren auch nicht so gut, jedoch ist mir Einheitlichkeit und das sich die Benutzer zurechtfinden wichtiger.

Reply:
Sehe ich auch so. Das macht am meisten Sinn in einem Rutsch umzubennen